pkgs/build-support/dart/list-dart-deps/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, dart
+, jq
+}:
+deps:
+
+builtins.fromJSON (builtins.readFile (stdenv.mkDerivation {
```

Comment:
It is used [here](https://github.com/NixOS/nixpkgs/blob/0495725a1f8e9df1ac3996459a9bb82305e69242/pkgs/build-support/flutter/default.nix#L112).

It is IFD, but it will generally only be evaluated right before a package build so I think it's all right. There's no easy way to do this without IFD.